logging/build.gradle.kts

```diff
@@ -30,8 +30,8 @@ kotlin {
             dependencies {
                 val ext = (gradle as ExtensionAware).extra
 
-                implementation("co.touchlab:stately-isolate:${ext["stately_version"]}")
-                implementation("co.touchlab:stately-iso-collections:${ext["stately_version"]}")
+                implementation("co.touchlab:stately-isolate:1.0.2-a4")
```

Comment:
why `1.0.2` elsewhere and `1.0.2-a4` here?

Reply:
Because these particular libraries are not in 1.0.2 yet, while the others are not in 1.0.2-a4. See https://github.com/touchlab/Stately/releases/tag/v1.0.2

Reply:
Please add a comment in the build file too.

Reply:
Done